sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuGenerateExec.scala

```diff
@@ -250,7 +250,7 @@ abstract class GpuExplodeBase extends GpuUnevaluableUnaryExpression with GpuGene
 
   override def fixedLenLazyExpressions: Seq[Expression] = child match {
     // GpuLiteral of ArrayData will be converted to GpuCreateArray with GpuLiterals
-    case GpuCreateArray(expressions, _) => expressions
+    case GpuCreateArray(expressions, __) => expressions
```

Comment:
Why do we have a `__` in here instead of a `_`?

Reply:
Done